### PR TITLE
cdo/180rc6

### PR DESCRIPTION
**Pull request by jgphpc**
https://webrt.cscs.ch/Ticket/Display.html?id=26342

### CODE REVIEW

File: jenkins-builds/6.0.UP02-2016.11-mc [hunk 1]
```diff
@@ -2,7 +2,7 @@
  Amber-16-2016.11-CrayGNU-2016.11-parallel.eb
  Boost-1.62.0-CrayGNU-2016.11-Python-3.5.2.eb
  Boost-1.61.0-CrayGNU-2016.11-Python-2.7.12.eb
- CDO-1.7.2-CrayGNU-2016.11.eb
+ CDO-1.8.0rc6-CrayGNU-2016.11.eb
```
Comment (gppezzi) [inline jenkins-builds/6.0.UP02-2016.11-mc]: Are you sure about removing the latest stable and replacing with a release candidate?

File: easybuild/easyconfigs/c/CDO/CDO-1.8.0rc6-nc_config.patch [hunk 2]
```diff
@@ -0,0 +1,11 @@
+diff -Naur cdo-1.8.0rc6.ori/configure cdo-1.8.0rc6/configure
+--- cdo-1.8.0rc6.ori/configure	2017-01-05 18:31:52.000000000 +0100
++++ cdo-1.8.0rc6/configure	2017-01-26 21:06:28.699660961 +0100
+@@ -19076,6 +19076,7 @@
+ 
+                                  NETCDF_LIBS=" -L$NETCDF_ROOT/lib -lnetcdf"
+                                  NETCDF_INCLUDE=" -I$NETCDF_ROOT/include"
++export NC_CONFIG=`which nc-config`
```
Comment (gppezzi) [inline easybuild/easyconfigs/c/CDO/CDO-1.8.0rc6-nc_config.patch]: Did you try to do this export using 'preconfigopts' ?

File: jenkins-builds/6.0.UP02-2016.11-mc (hunk 1 above)
Comment (jgphpc) [inline jenkins-builds/6.0.UP02-2016.11-mc]: no, i will put it back.

File: easybuild/easyconfigs/c/CDO/CDO-1.8.0rc6-nc_config.patch (hunk 2 above)
Comment (jgphpc) [inline easybuild/easyconfigs/c/CDO/CDO-1.8.0rc6-nc_config.patch]: i tried, no success so far.

**Comment by jgphpc** [comment]
* [mc ok](https://jenkins.cscs.ch/view/ProductionEB/job/ProductionEBDaint-mc/77/label=daint/console)
* [gpu ok](https://jenkins.cscs.ch/view/ProductionEB/job/ProductionEBDaint-gpu/165/label=daint/console)

**Pull request merged**